### ISSUE TITLE
async_stop_inclusion KeyError

### ISSUE THREAD

**Issue by cgarwood** [home-assistant-libs/zwave-js-server-python#47]
If an inclusion is running, firing async_stop_inclusion gives the following error:
```
Traceback (most recent call last):
  File "/home/cgarwood/home-assistant/core/homeassistant/components/websocket_api/decorators.py", line 18, in _handle_async_response
    await func(hass, connection, msg)
  File "/home/cgarwood/home-assistant/core/homeassistant/components/zwave_js/websocket_api.py", line 91, in websocket_stop_inclusion
    result = await controller.async_stop_inclusion()
  File "/home/cgarwood/home-assistant/core/venv/lib/python3.8/site-packages/zwave_js_server/model/controller.py", line 152, in async_stop_inclusion
    return cast(bool, data["success"])
KeyError: 'success'
```

If an inclusion is not running, it returns False as expected.

cc @Michsior14

**Comment by Michsior14** [comment]
@cgarwood  Could you see any error logged on `zwave-js-server` side?

**Comment by cgarwood** [comment]
zwave-js-server console output didn't log any errors. Just showed that the controller entered inclusion mode and that inclusion mode stopped when I hit the stop button.

**Comment by Michsior14** [comment]
It seems that `zwave-js` responses with `undefined` in command `stopInclusion` which causes the error. I will check the sources to find out how we can prevent it ;)

**Comment by Michsior14** [comment]
The PR with fix on `zwave-js` side has been created - https://github.com/zwave-js/node-zwave-js/pull/1412.

**Comment by cgarwood** [comment]
Should be fixed in zwave-js 6.0.0.beta3 which is now the default for zwave-js-server
